src/rng/test/util_cpu.h

```diff
@@ -39,170 +39,183 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  * before reading /proc/cpuinfo, might fool
  * energy-saving CPU into showing its true speed
  */
-static double warmupCPU(long n){
-    double d = 0.3;
-    int i;
-    for (i = 0; i < n; i++) {
-	d = 3.6 * d * (1. - d);
-    }
-    dprintf(("logistic map produced %f\n", d));
-    return d;
+static void warmupCPU(long n) {
+  double d = 0.3;
+  int i;
+  for (i = 0; i < n; i++) {
+    d = 3.6 * d * (1. - d);
+  }
+  dprintf(("logistic map produced %f\n", d));
 }
 
 #if defined(_MSC_VER)
-#define NOMINMAX    /* tells Windows.h to NOT define min() & max() */
+#define NOMINMAX /* tells Windows.h to NOT define min() & max() */
 #include <Windows.h>
 static double clockspeedHz(int *ncores, char **modelnamep) {
-    /*
+  /*
      * To-Do: one could read clock speed from registry.  Or
```

Comment:
@RyanWollaeger Can you fix the indentation for these comment blocks so that match the first line (`/*`).  It looks like clang-format is doing something unexpected here.

Reply:
@KineticTheory Appologies and thank you for catching this - this will be corrected shortly.